pom.xml

```diff
@@ -79,7 +79,7 @@
 
   <properties>
     <mojo.java.target>1.7</mojo.java.target>
-    <mavenVersion>3.5.2</mavenVersion>
+    <mavenVersion>3.8.3</mavenVersion>
```

Comment:
It should be safe to update to the latest version on 3.x.x:
`<mavenVersion>3.9.4</mavenVersion> `